extensions/amp-a4a/0.1/test/test-callout-vendors.js

```diff
@@ -0,0 +1,11 @@
+import {RTC_VENDORS} from '../callout-vendors';
+
+// The keys of RTC_VENDORS are not allowed to have any capital letters.
+// This test acts as a presubmit to enforce that.
+describe('RTC_VENDORS', () => {
+  it('should have all lowercase keys', () => {
+    Object.keys(RTC_VENDORS).forEach(key => {
```

Comment:
code golf:  Object.key(RTC_VENDORS).forEach(key => expect(key).to.equal(key.toLowerCase()));

Optionally if you know what non-alphanumeric characters are allowed you could use a regexp: expect(key).to.match(/[0-9a-z_]+/)